src/components/navigation/nav-items/nav-items.jsx

```diff
@@ -0,0 +1,139 @@
+import { React } from "react";
+import { Nav } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  // faTachometerAlt,
```

Comment:
Why this?

Reply:
It's related to the dashboard. I have updated the comment.